WooCommerce/Classes/ViewRelated/Products/ProductListSelector/LegacyProductListSelectorViewController.swift

```diff
@@ -0,0 +1,183 @@
+import Combine
+import UIKit
+import Yosemite
+
+/// Displays a paginated list of products where the user can select.
```

Comment:
nit: It could be useful to explain why now it's called Legacy, and that the plan is to remove it in the near future 👍

Reply:
Updated in 4130eb3731527d40fa0fa26800c989b76f630b22.